marimo/_server2/api/deps.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 Marimo. All rights reserved.
+from __future__ import annotations
+
+from typing import Annotated, Literal, Optional
```

Comment:
`Annotated` is new in Python 3.9. For Python 3.8 compatibility, `Annotated` must be imported from `typing_extensions`